Gemfile.lock

```diff
@@ -1,3 +1,12 @@
+GIT
```

Comment:
This stuff will be dependent on whether you go with just `thumbs_up` or a specific revision of the project (see comment on Gemfile line 12)... if you do go with the generic, make sure you run bundler again so the Gemfile.lock is updated.